src/actions.js

```diff
@@ -0,0 +1,32 @@
+export const inferResponseLimit = function(response) {
+  const count = response.count;
+  const resultCount = response.results.length;
+
+  if (count > resultCount) {
+    return resultCount;
+  }
+
+  return null;
+};
+
+export const parseResponse = function(response) {
```

Comment:
I called it parse since it's only pulling out the properties of the response it cares about, and creating a new object.

However, I do see where you're coming from. The function is really only calculating the `pageCount`. I'll refactor it to do only that.